src/main/java/servlet/PostServlet.java

```diff
@@ -62,8 +62,11 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
             {
                 /** MODIFIED BY STEFAN SO ONLY AUTHORIZED USERS CAN DELETE POSTS */
                 HttpSession session=request.getSession(false);
+                String uname = (String)session.getAttribute("username");
+                ArrayList<String> n = (ArrayList<String>)session.getAttribute("membership");
+                String groupName = n.get(0);
```

Comment:
Why are you using index 0? What happen if the user have multiple groups? I would suggest passing all groups in and compare the group membership with the group of the post

Reply:
I assumed if a user was admin, it would indicate it at index 0. 
I will change that so it actually checks if 'admins' exists in the membership session.

Reply:
changes are done!